R/server.R

```diff
@@ -2123,6 +2131,35 @@ AzimuthServer <- function(input, output, session) {
       }
     }
   )
+  output$dlall <- downloadHandler(
+    filename = paste0(tolower(x = app.title), '_all.Rds'),
```

Comment:
Maybe we call it `azimuth_results.Rds` so it's consistent with the reader function (`AddAzimuthResults`)?